src/freenas/usr/local/lib/collectd_pyplugins/ifstat.py

```diff
@@ -0,0 +1,63 @@
+import traceback
+
+import collectd
+from pyroute2 import IPRoute
+
+from middlewared.client import Client
+
+READ_INTERVAL = 1.0
```

Comment:
Isn't every second too often? `interface` plugin files had a standard step of 10 seconds.